### ISSUE TITLE
Support for Python 3.11

### ISSUE THREAD

**Issue by grizzlybearg** [man-group/dtale#771]
Hi, is there a timeline for support for python 3.11?

**Comment by aschonfeld** [comment]
So I've tried to get the ball rolling on this but I'm hitting an error when trying to run my tests.

```
(dtale_311) Andrews-MacBook-Pro:dtale andrewschonfeld$ python setup.py test
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing dtale.egg-info/PKG-INFO
writing dependency_links to dtale.egg-info/dependency_links.txt
writing entry points to dtale.egg-info/entry_points.txt
writing requirements to dtale.egg-info/requires.txt
writing top-level names to dtale.egg-info/top_level.txt
reading manifest file 'dtale.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'dtale.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "/Users/andrewschonfeld/dev/git/dtale/setup.py", line 73, in <module>
    setup(
    ^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/envs/dtale_311/lib/python3.11/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/lib/python3.11/distutils/core.py", line 148, in setup
    dist.run_commands()
    ^^^^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/lib/python3.11/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/lib/python3.11/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
    ^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/envs/dtale_311/lib/python3.11/site-packages/setuptools/command/test.py", line 232, in run
    self.run_tests()
    ^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/dev/git/dtale/setup.py", line 48, in run_tests
    import pytest
    ^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/envs/dtale_311/lib/python3.11/site-packages/pytest/__init__.py", line 8, in <module>
    from _pytest.capture import CaptureFixture
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/envs/dtale_311/lib/python3.11/site-packages/_pytest/capture.py", line 587, in <module>
    class CaptureResult(NamedTuple, Generic[AnyStr]):
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/andrewschonfeld/.pyenv/versions/3.11.0a1/lib/python3.11/typing.py", line 2301, in _namedtuple_mro_entries
    raise TypeError("Multiple inheritance with NamedTuple is not supported")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Multiple inheritance with NamedTuple is not supported
```

Any chance you have any insight around how I can get around these TypeErrors?

**Comment by aschonfeld** [comment]
Ok the problem was that I was using an old version of python 3.11, I needed to upgrade to 3.11.4.  Will keep you posted on the progress of this...

**Comment by grizzlybearg** [comment]
Thanks for the update

**Comment by aschonfeld** [comment]
@grizzlybearg just released v3.2.0 to pypi (conda-forge coming shortly) with support for 3.11.  Let me know how it goes.

Also, if you haven't already, please put your ⭐ on the repo when you get a sec. Thanks! 🙏

**Comment by grizzlybearg** [comment]
Thanks
